test/no-imporant_test.js

```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai';
+import aphroditeInterface from '../src/no-important';
+
+describe('no-important', () => {
+  it('is an interface', () => {
+    expect(typeof aphroditeInterface.create).to.equal('function');
+    expect(typeof aphroditeInterface.resolve).to.equal('function');
+  });
```

Comment:
It might be worth adding a test here that asserts that `!important` is not used, and another test on the other one that asserts the opposite.